src/praisonai/praisonai/cli/main.py

```
@@ -351,6 +351,18 @@ def run(self):
         """
         return self.main()
 
+    @staticmethod
+    def _require_agents():
+        """Exit with an install hint if praisonaiagents is unavailable.
+
+        Single source of truth for the "needs praisonaiagents" guard so the same
+        check is not duplicated across every command branch.
+        """
+        if not PRAISONAI_AVAILABLE:
```

Ruff (0.15.18)

[error] Undefined name `PRAISONAI_AVAILABLE`

(F821)